test/extensions/filters/listener/http_inspector/http_inspector_test.cc

```diff
@@ -42,6 +42,7 @@ class HttpInspectorTest : public testing::Test {
     EXPECT_CALL(socket_, detectedTransportProtocol()).WillRepeatedly(Return("raw_buffer"));
     EXPECT_CALL(cb_, dispatcher()).WillRepeatedly(ReturnRef(dispatcher_));
     EXPECT_CALL(socket_, ioHandle()).WillRepeatedly(ReturnRef(*io_handle_));
+    EXPECT_CALL(testing::Const(socket_), ioHandle()).WillRepeatedly(ReturnRef(*io_handle_));
```

Comment:
Do we need both const and non-const variants?

Reply:
updated so that it isn't needed anymore.